elastic-agent.yml

```diff
@@ -38,6 +39,18 @@ inputs:
         - filesystem
         data_stream.dataset: system.filesystem
 
+  # Collecting log files
+  #- type: filestream
+  #  id: your-input-id:
```

Comment:
This file looks like it needs to be re-templated, the colon is still here.

Reply:
Fixed. I used the "merge suggestion" feature but didn't pull it down and run `check`. Sorry for the extra round trip.